### PR TITLE
translation to ES of review.md

### LINKED ISSUE #301: Create spanish review template
To document my review in Spanish as requested by @melvidoni I looked for a spanish language template. As I could not find it I made a translation.

I decided to name the file `review-es.md` and not a translated version of the file name so that the file content is still clear for a non-spanish speaker (`revision.md` would no be so hard to identify, but in other languages in the future this may be different).

### PR DESCRIPTION
**Pull request by FvD**
Fix #301 

- Pull request to dev branch as per request of @maelle in #302

### CODE REVIEW

**Comment by maelle** [comment]
:pray: muchas gracias

**Comment by maelle** [comment]
I'll add the template to the book (via editing one Rmd) myself once the review has been approved by one native speaker. Thanks a ton again!

File: templates/review-es.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+>
+>- [ ] **A short summary** describing the high-level functionality of the software
+>- [ ] **Authors:**  A list of authors with their affiliations
+>- [ ] **A statement of need** clearly stating problems the software is designed to solve and its target audience.
+>- [ ] **References:** with DOIs for all those that have one (e.g. papers, datasets, software).
+
+#### Funcionalidad
+
+- [ ] **Instalación:** La instalación se completa con éxito tal como fue documentado.
+- [ ] **Funcionalidad:** Todo afirmación de funcionalidad del software se confirma como existente.
```
Comment (maurolepore) [inline templates/review-es.md:34]: > - [ ] **Funcionalidad:** [Todo]^1 afirmación de funcionalidad del software se confirma como existente.

Instead I suggest the feminine [Toda] (modifies the feminine "la afirmación").

File: templates/review-es.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+- [ ] **Desempeño:** Toda afirmación de desempeño del software se confirma como alcanzada.
+- [ ] **Pruebas automáticas:** Hay pruebas unitarias que cubren las funciones esenciales dentro del paquete con un rango razonable de entradas y condiciones. Todas las pruebas corren en la maquina local.
+- [ ] **Directrices de empaque**: El paquete cumple con las directrices de empaque de rOpenSci.
+
+Estimación de horas dedicadas a la revisión:
+
+- [ ] Si la o las persona(s) autora(s) lo considera(n) apropiado, yo estoy de acuerdo con que me reconozcan como revisor del paquete (el rol "rev') en la el archivo DESCRIPTION del paquete.
```
Comment (maurolepore) [inline templates/review-es.md:41]: I think it's correct. 

An alternative might be:

- [ ] Si me lo ofrecieran, acepto el rol "rev" en la el archivo DESCRIPTION del paquete.

**Review by maurolepore — CHANGES_REQUESTED** [review]
Thanks a lot @FvD! 

In addition to the changes I suggest here, I also suggest some changes to the English version (@maelle I mentioned you in a few places -- let me know if you want me to follow up with a PR). Also, I think both versions would benefit from wrapping at 80 characters -- that make it easier for a reviewer to see exactly what changed when someone fixes a typo.

File: templates/review-es.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
+## Revisión de un paquete
+
+*Por favor trata de marcar tantas casillas como te es posible y elabora tus argumentos en comentarios abajo de cada uno. Tu revisión no se necesita listar a estos temas, tal como se describe en la guia para revisores (Reviewer Guide)*
+
+**Por favor describe cualquier relación de trabajo que tienes/has tenido con los autores del paquete)**
```
[the remark this answers is not in the document]
Comment (maelle) [reply templates/review-es.md:5]: Well we have two editors from Argentina so fine to use this form IMHO. :grin:

File: templates/review-es.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+- [ ] Como revisor confirmo que no tengo [conflictos de interés](https://devguide.ropensci.org/policies.html#coi) para poder hacer la revisión de este trabajo (si no estas segura si tienes un conflicto por favor entra en contacto con tu editor _antes_ de arrancar con la revisión.
+
+#### Documentación
+
+El paquete incluye todos los siguiente tipos de documentación:
+
+- [ ] **Una declaración de necesidades** que claramente describe las necesidades que el software esta diseñado a resolver y el public meta que busca atender en el archivo README
+- [ ] **Instrucciones de instalación** tanto para la versión en desarrollo incluyendo cualquier dependencia no-estándar en el archivo README
+- [ ] **Vignette(s)** demostrando la funcionalidad principal que ademas corren localmente
+- [ ] **Documentación de las funciones** en la ayuda de R para todas la funciones exportadas
```
[the remark this answers is not in the document]
Comment (maelle) [reply templates/review-es.md:16]: makes sense!

**Comment by maelle** [comment]
Thanks @FvD & @maurolepore!

**Comment by maelle** [comment]
(changes made in the English version too)

**Pull request merged**